src/client/src/views/SceneForm.js

```diff
@@ -8,7 +8,7 @@ module.exports = {
   oninit: function(vnode) {Scene.setCurrent(vnode.attrs.id - 1)},
   view: function(vnode) {
     const captions = Scene.current.captions
-    
+    //console.log(captions)
```

Comment:
Commented out console logs like this should not have been included in the commit.

Otherwise looks good! Thank you for writing tests as well.

Reply:
I went ahead and removed that console log (as well as another one that snuck its way in previously). 

So now this PR is ready to be approved and merged in. But now I since I added to it I think someone else has to approve.